doc/tutorials/using_celadon_as_uos.rst

```diff
@@ -0,0 +1,113 @@
+.. _using_celadon_as_uos:
+
+Using Celadon as User OS
+########################
+
+`Celadon <https://01.org/projectceladon/>`_ is an open source Android* software reference stack
+for Intel architecture. It builds upon a vanilla Android stack and incorporates open sourced components
+that are optimized for the hardware. This tutorial describes how to run Celadon as the User OS
+on the ACRN hypervisor. We are using Kaby Lake-based NUC (model NUC7i7DNHE) in this tutorial.
+Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
+Introduction to OVMF
+********************
+
+Open Virtual Machine Firmware (OVMF) is an `EDK II
+<https://github.com/tianocore/tianocore.github.io/wiki/EDK-II>`_
+based project to enable `UEFI <https://github.com/tianocore/tianocore.github.io/wiki/UEFI>`_
+support for Virtual Machines. OVMF contains sample UEFI firmware for `QEMU <http://www.qemu-project.org/>`_ and `KVM
+<http://www.linux-kvm.org/>`_.
+
+Prerequisites
+*************
+
+* Ubuntu 16.04 or higher with at least 150G free disk space
```

Comment:
We would like to get rid of Ubuntu 16.04 (there are too many problems if you're trying to build ACRN on it), can we refer to Ubuntu 18.04 or higher instead? Is it known to work for a fact?

Reply:
Thanks @ttzeng ! @lirui34 , can you update this tutorial to only reference Ubuntu 18.04 and above?